src/ResourceManager/Billing/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Current Release
 
+## Version 0.15.0
```

Comment:
Please remove the version heading, these changes should just be in 'Current Release'

Reply:
Fixed